redacted.gemspec

```diff
@@ -18,10 +18,10 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "ffaker", "~> 2.3"
-  spec.add_dependency "nokogiri", "~> 1.6"
-  spec.add_dependency "railties", ">= 3.2", "< 5.0"
+  spec.add_dependency "ffaker", "~> 2.8.1"
+  spec.add_dependency "nokogiri", "~> 1.8.2"
+  spec.add_dependency "railties", ">= 3.2", "< 6.0"
 
   spec.add_development_dependency "bundler", "~> 1.6"
-  spec.add_development_dependency "rails", "~> 4.1"
+  spec.add_development_dependency "rails", "~> 5.1"
```

Comment:
Can we simply make this `>= 4.1` to maintain 4.1 support?

Reply:
That makes sense!